lib/m2ts/caption-stream.js

```diff
@@ -139,17 +140,29 @@ var parseCaptionPackets = function(pts, userData) {
   return results;
 };
 
+var activeChannel = null;
```

Comment:
After reading your comment about the difficulty testing this because it's (module) global, I realized that this is going to have to change.

The goal is to have some state that is shared between all 4 `Cea608Stream` objects. Here is what I am thinking: 
* Make the `activeChannel` state a property of the `CaptionStream`
* Have the `Cea608Stream` objects emit an event like "activechannelchange" that alerts the `CaptionStream` object that it needs to change the `activeChannel`
* Have the `CaptionStream` call each `Cea608Stream#setActiveChannel` to update each stream with the new active channel information.